### PR TITLE
Add removefromconsult and resolve existing conflicts

### PR DESCRIPTION
**Pull request by marcusjhang**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 1]
```diff
@@ -64,6 +66,9 @@ public Command parseCommand(String userInput) throws ParseException {
         case AddConsultCommand.COMMAND_WORD:
             return new AddConsultCommandParser().parse(arguments);
 
+        case RemoveFromConsultCommand.COMMAND_WORD:
+            return new RemoveFromConsultCommandParser().parse(arguments);
```
Comment (notnotmax) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java:70]: Consider adding a testcase?

**Review by notnotmax — CHANGES_REQUESTED** [review]
Looks good, just missing a testcase like your other PR.

**Review by notnotmax — APPROVED** [review]
lgtm, ready for merge.

**Review by S-K-Y-Light — COMMENTED** [review]
LGTM

**Pull request merged**